bridge/client/app/_components/ktb-project-tile/ktb-project-tile.component.html

```diff
@@ -1,20 +1,20 @@
 <dt-tile [id]="project.projectName" [routerLink]="['/project', project.projectName]" *ngIf="project">
-  <dt-tile-title [textContent]="project.projectName" [class.error]="project.isShipyardNotSupported(supportedShipyardVersion)"></dt-tile-title>
+  <dt-tile-title [textContent]="project.projectName" [class.error]="project.isShipyardNotSupported(supportedShipyardVersion)" uitestid="keptn-project-tile-title"></dt-tile-title>
   <dt-tile-subtitle>
-    <p class="mt-0 mb-2" *ngIf="project.stages"><span [textContent]="project.stages.length"></span> Stages, <span [textContent]="project.getServices().length"></span> Services</p>
-    <p class="mt-0 mb-2" *ngIf="project.getShipyardVersion() as sv">Shipyard version: <span [textContent]="sv"></span> <dt-icon name="abort" class="error bottom" *ngIf="project.isShipyardNotSupported(supportedShipyardVersion)"></dt-icon> </p>
+    <p class="mt-0 mb-2" *ngIf="project.stages" uitestid="keptn-project-tile-numStages"><span [textContent]="project.stages.length"></span> Stages, <span [textContent]="project.getServices().length"></span> Services</p>
```

Comment:
Suggestion:
`<p class="mt-0 mb-2" *ngIf="project.stages"><span uitestid="keptn-project-tile-numStages" [textContent]="project.stages.length"></span> Stages, <span  uitestid="keptn-project-tile-numServices" [textContent]="project.getServices().length"></span> Services</p>`

Reply:
I have been told to not put it on lower level elements, but on containers, therefore I wanted to avoid putting it directly on a span.

Reply:
Hm tricky :D it's both
`keptn-project-tile-numStagesServices` ?

Reply:
Done